common-util/Login/LoginV2.tsx

```diff
@@ -129,7 +129,7 @@ export const LoginV2 = ({ onConnect: onConnectCb, onDisconnect: onDisconnectCb }
         </YellowButton>
       )}
       &nbsp;&nbsp;
-      <w3m-button balance={isMobile ? 'hide' : 'show'} />
+      <w3m-button balance={'hide'} />
```

Comment:
it's very dragging while switching chains, will plan fixing this for next weeks and return back how it was